compiler/rustc_middle/src/mir/pretty.rs

```diff
@@ -944,7 +944,7 @@ impl<'tcx> Debug for Rvalue<'tcx> {
                     NullOp::SizeOf => write!(fmt, "SizeOf({t})"),
                     NullOp::AlignOf => write!(fmt, "AlignOf({t})"),
                     NullOp::OffsetOf(fields) => write!(fmt, "OffsetOf({t}, {fields:?})"),
-                    NullOp::UbCheck(kind) => write!(fmt, "UbCheck({kind:?})"),
+                    NullOp::UbChecks => write!(fmt, "UbChecks()"),
```

Comment:
I don't see any corresponding changes to the mir-opt tests, which has me worried. CI is green but the mir-opt suite does not pass locally.

Reply:
Good point. I think this happens because of
```
//@ ignore-debug assertions change the output MIR
```
The PR runner has debug assertions turned on.

Reply:
Ah yes this old problem. I'll see if I can do a PR to fix it.

Reply:
Is that even still needed? The stdlib is less dependent on the sysroot `cfg(debug_assertions)` now with your new intrinsic.

Reply:
There is _less_ need for it, but it's not gone entirely. I'll start by just enabling the tests that can be enabled for free: https://github.com/rust-lang/rust/pull/122921